.changeset/proud-cows-jog.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sumup/circuit-ui': minor
```

Comment:
This would technically be breaking for Circuit unfortunately (just like Chromatic says in the PR), because apps would be forced to bump `@sumup/icons` to have the `NotifyTriangle` icon available.

However I remember talking with @connor-baer about bypassing this rule for our own dependencies, and clarify that apps should always bump SumUp packages together. WDYT?

Otherwise, an alternative would be to keep the `Notify` name for the triangle variant and name the new variant `NotifyCircle` instead (so Circuit doesn't have a version bump at all).

([this new notify icon seems weird to me anyways](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC?node-id=4135:19#121333108))

Reply:
We need the circular variant for the Chargeback events in Sales history.
<img width="574" alt="Screenshot 2021-11-02 at 17 23 23" src="https://user-images.githubusercontent.com/9676303/139877233-727065dd-d2a0-4570-8825-e2ded7acf1bc.png">

To me the triangular version is the exception to the rule, since all the rest of the status icons are circles 😄 I don't particularly like the idea of having two variants of this icon as well. I'm perfectly OK to go the `NotifyCircle` path if this is the less common variant in terms of usage though.

Reply:
Yes, the rule we want to follow in the Circuit UI ecosystem is that required minor and patch updates in peer dependencies don't require a major version of the parent dependency.

This should be documented in the `MIGRATION.md` file though to avoid confusion.

Reply:
According to the discussion in Figma the triangular variant should be the main one, so I ended up renaming the circular variant to `NotifyCircle` as Robin suggested. This means that no version bump of Circuit and no migration instructions are necessary.